src/autobot/autobottypes.h

```diff
@@ -26,16 +26,9 @@
 #include "iteststep.h"
 
 namespace mu::autobot {
-enum class FileStatus {
-    Undefined = 0,
-    None,
-    Success,
-    Failed
-};
-
 struct File {
     io::path path;
-    FileStatus status = FileStatus::Undefined;
+    Ret ret;
```

Comment:
Could you explain why you need ret here?

Reply:
this is the result of the file test (in the future, there may be code and text message)

Reply:
I would prefer to rename this to "testCaseRet" then :) Or rename this struct to FileTestResult (or something similar on your choice :))